### DISCUSSION TITLE
Problem with Simple Experiment Tutorial

### DISCUSSION THREAD

**Discussion by dimitrats94** [l3atbc-datadog/pushkin#171] [Q&A] [answered]
Hello everyone! 

My experience with coding is close to none so I hope my question is not too silly. 

I have installed Pushkin and dependencies in order to integrate a script written with JsPsych. I successfully completed all steps until Quickstart and now I'm trying to run the Simple Experiment Tutorial that you provide. I followed each step carefully, but I came across an error that I can't seem to resolve after $pushkin start;

The message that I get is the following: 

"The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing. 
Continue with the new image? [yN]"

I've tried responding with yes or YES or y but it doesn't move past this. 

Could you help me out? What is wrong in your opinion?

Thanks in advance!

**Comment by restmegan** [comment, marked as answer]
Hi @dimitrats94 ! I would suggest running `pushkin armageddon` in your experiment, then running `pushkin prep` again before trying `pushkin start`. Let us know if this works for you!

**Reply by Weiiirrr** [reply]
I have the same issue here. But this solution didn't work for me.

**Comment by jkhartshorne** [comment]
@dimitrats94 did this work for you?

**Reply by colinfindley** [reply]
I'm having an issue with pushkin prep while following the simple tutorial. I'm using pushkin 3.1. I completed the quickstart and was able to see the preview via localhost. I added "lex" and copied the jspsych code into the relevant js file under /src.  Whenever I try to prep, I run into this error message. 

"TypeError: Cannot read property 'webPage' of undefined"

**Reply by Weiiirrr** [reply]
I figured this out! It's because of `.DS_store`. Check [this](https://github.com/pushkin-consortium/pushkin/discussions/88) out. Hope it helps!
